Overview.src.html

```diff
@@ -2663,6 +2685,30 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <p><span title="Read a request">Read <var>request</var></span>.
 
  <li>
+  <p>Let <var>strategy</var> be an object. The user agent may choose any object.
+
+  <p class="note no-backref"><var>strategy</var> is used to control the queuing strategy of
+  <var>stream</var> constructed below.
+
+ <li><p>Let <var>pull</var> be an action that <span title=concept-fetch-resume>resumes</span> the
```

Comment:
Oh, I see, the "function" part is taken care of by the "construct a ReadableStream" algorithm. Nevermind!